code/modules/clothing/suits/cloaks.dm

```diff
@@ -89,3 +89,41 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/neck/cloak/skill_reward
+	var/associated_skill_path = /datum/skill
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE
+
+/obj/item/clothing/neck/cloak/skill_reward/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You notice a powerful aura about this cloak, suggesting that only the truly experienced may wield it.</span>"
+
+/obj/item/clothing/neck/cloak/skill_reward/equipped(mob/user, slot)
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
Could you make it not be equippable at all, instead of equipped and dropped?

Reply:
donesies

Reply:
The suggestion still stands.

Reply:
yeah i'm retarded, my repo is shitting itself due to the whole github thing. please hold